lib/storage/metadata/mongoclient/readStream.js

```diff
@@ -63,9 +63,13 @@ class MongoReadStream extends Readable {
             Object.assign(query, searchOptions);
         }
 
-        this._cursor = c.find(query).sort({
-            _id: options.reverse ? -1 : 1,
-        });
+        this._cursor = c.find(query);
+
+        if (!searchOptions) {
```

Comment:
I assume we sort when there are no search options provided because we can consider that sorting by `_id` is always efficient? Could deserve a comment since it does not look obvious.